ansible/maintenance.yml

```diff
@@ -1,25 +1,54 @@
 ---
+# Check out the branch first as that tells us what snapshot version is being updated
+# This is necessary to set baseline_branch
+- name: Checkout branches from GitHub
+  hosts: performance-candlepin-vm
+  roles:
+    - candlepin-checkout
+  vars:
+    ansible_user: jenkins
+  tags:
+    - checkout
+
```

Comment:
What about situation, when one or both VMs are shutdown and current image is corrupted for some reason (testing, experiments). I suspect that we would lost existing `readyfortestsnap` snapshot and we would create new useless.

I propose:
1. Force shutdown all VMs
2. Kickstart VMs from snapshots to make sure VMs are in good condition
3. Do necessary maintenance in VMs
4. Shutdown all VMs
5. Create new snapshots